.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: node_js
+
+git:
+  depth: 1
+
+branches:
+  only:
+    - master
+    - /^[0-9]+\.[0-9]+\.[0-9]+.*/
```

Comment:
Let's go ahead and add a rule for `rc-` branches, too.
```
- /^rc-.*$/
```

Reply:
Done.